src/index.html

```diff
@@ -22,6 +22,336 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header>
+      <div class="logo">
+        <a
+          class="logo__link"
+          href="#"
+        >
+          <img
+            class="logo__img"
+            src="./images/logo.png"
+            alt="logo"
+          />
+        </a>
+      </div>
```

Comment:
According to the BEM methodology, block elements should not be nested. The `logo` block should be a standalone component, and the `logo__link` should be an element of the `logo` block without additional nesting.